src/data/events.js

```diff
@@ -30,7 +30,9 @@ export async function create(event) {
     emails: event.emails,
   })
 
-  events.update(currentEvents => [newEvent, ...currentEvents])
+  loadEvents()
```

Comment:
this call to `loadEvents` puts more load on the server, so if it's not necessary it's better to leave it as it was

Reply:
I put this here because, for some reason, after you create an event, the newly created event appears at the top, but when you reload the events page it goes in the right order. I assume the order is fetched from the server.